client/components/Guests.jsx

```diff
@@ -1,26 +1,38 @@
 import React from 'react';
 import GuestDropdown from './GuestDropdown.jsx';
-import { BoldText, BoxText, BoxIcon, BoxWrapper, GuestMenuBtn } from './styled/Styled.jsx';
+import { GuestsStyle, BoldText, BoxText, BoxIcon, BoxWrapper, GuestMenuBtn } from './styled/Styled.jsx';
```

Comment:
again, i'm assuming the naming here of styled components takes into account the scope of their use. BoxText is pretty broad, for example.